Frontend/linkedout/src/main/pages/ProfilePage/ProfilePageJob/ProfilePageJob.tsx

```diff
@@ -46,49 +52,81 @@ export const ProfilePageJob = () => {
 
     const handleDeleteExperienceDialog = () => {
         setOpenDialog(false);
-        if(isCandidate)
-            setExperience(experience.slice(0, currentExperience).concat(experience.slice(currentExperience + 1, experience.length)));
-        else
+        if(isCandidate){
+            // setExperience(experience.slice(0, currentExperience).concat(experience.slice(currentExperience + 1, experience.length)));
+            delete_experience(bufferId);
+        }       
+        else{
             setJobPostings(jobPostings.slice(0, currentExperience).concat(jobPostings.slice(currentExperience + 1, jobPostings.length)));
+        }
+        window.location.reload();
```

Comment:
Why is this needed? Technically you're updating the state and you shouldn't need to manually reload the page

Reply:
I have decided to make the experience show in chronological order as it would make more sense. Im doing it in the backend. Out of laziness, I just refresh the page instead of relying on the frontend.